dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild

```diff
@@ -0,0 +1,287 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit flag-o-matic eutils autotools multilib user readme.gentoo versionator
+
+MY_P=${PN/f/F}-$(replace_version_separator 4 -)
+#MY_P=${PN/f/F}-${PV/_rc/-ReleaseCandidate}
+
+DESCRIPTION="A relational database offering many ANSI SQL:2003 and some SQL:2008 features"
+HOMEPAGE="http://www.firebirdsql.org/"
+SRC_URI="
+	mirror://sourceforge/firebird/${MY_P}.tar.bz2
+	 doc? (	ftp://ftpc.inprise.com/pub/interbase/techpubs/ib_b60_doc.zip )"
+
+LICENSE="IDPL Interbase-1.0"
+SLOT="0"
+KEYWORDS="~amd64 -ia64 ~x86"
+
+IUSE="doc client superserver xinetd examples debug"
+REQUIRED_USE="
+	client? ( !superserver )
+	client? ( !xinetd )
+	superserver? ( !xinetd )
```

Comment:
Most these USE flags are going away on next major version. The client USE flag I never really got to finish. It can be improved in many ways.